### PR TITLE
[rappture] Update port to use github

### PR DESCRIPTION
**Pull request by AenBleidd**
Signed-off-by: Vitalii Koshura <lestat.de.lionkur@gmail.com>

### CODE REVIEW

**Comment by Hoikas** [comment]
What's wrong with the official source?

File: ports/rappture/portfile.cmake [hunk 1]
```diff
@@ -1,28 +1,24 @@
...
-    URLS "https://nanohub.org/app/site/downloads/rappture/rappture-src-20130903.tar.gz"
-    FILENAME "rappture-src-20130903.tar.gz"
-    SHA512 3b42569d056c5e80762eada3aff23d230d4ba8f6f0078de44d8571a713dde91e31e66fe3c37ceb66e934a1410b338fb481aeb5a29ef56b53da4ad2e8a2a2ae59
-)
-
-vcpkg_extract_source_archive_ex(
+vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
-    ARCHIVE ${ARCHIVE}
+    REPO AenBleidd/rappture
```
Comment (JonLiu1993) [inline ports/rappture/portfile.cmake:5]: @AenBleidd, I can’t find AenBleidd/rappture REPO on github,

**Comment by AenBleidd** [comment]
> What's wrong with the official source?

There was several times recently when server was not operating normally, and thus all builds fail.
GitHub is definitely more stable in this case.

**Review by JonLiu1993 — APPROVED** [review]

**Comment by Hoikas** [comment]
I think there are better ways to work around questionable availability (eg using asset caching) than redirecting a port in the official catalog to your own personal fork.

**Comment by AenBleidd** [comment]
@Hoikas, that's a questionable statement. At first, there is no actually be development on this library. Second, I can keep my fork (that is actually not a fork but mirror) always up-to-date with the original SVN repository.
But if you strongly against - I can simply close my PR

**Comment by AenBleidd** [comment]
@JonLiu1993, may I ask you to review this PR again and put 'reviewed' label? Thanks in advance

**Comment by JonLiu1993** [comment]
@AenBleidd ,Sprry for reply you so late, as @Hoikas said ,We do not recommend changing the official library to your own private library, and searching AenBleidd/rappture repo on google has no search results.

**Comment by AenBleidd** [comment]
Ok, no problem. I'll then open a new PR with to convert CONTROL to vcpkg.json

**Pull request closed without merging**